docs/doxygen/migration/jobs.dox

```diff
@@ -0,0 +1,35 @@
+/**
+@page jobs_migration AWS IoT Jobs
+@brief How to migrate an application using AWS IoT Jobs from v3 to 202009.00.
+
+The AWS IoT Jobs client library has been refactored for this release.
+The refactored library has the following differences as compared to the jobs client library in v3 of the AWS IoT Device SDK for Embedded C:
+
+- The refactored library provides simple convenience macros and functions for handling MQTT topic strings as defined by the AWS IoT Jobs Service.
+- The refactored library is completely decoupled from the underlying MQTT implementation, permitting any MQTT client library to be used.
+- The refactored library does not construct JSON message bodies, permitting any JSON library to be used.
+
+The v3 jobs client library, on the other hand, is tightly coupled to the MQTT implementation present in v3 of the AWS IoT Device SDK for Embedded C.
+It consists of a heavyweight set of APIs with multiple dependencies.
+JSON parsing, for example, is built in to the API implementation.
+
+Note: The AWS IoT Device SDK for Embedded C now offers the coreMQTT library and the coreJSON library as supported alternatives.
+
+To migrate an existing application
+- change the setup code to establish a connection to AWS IoT core
+- add a subscription step to the setup code
```

Comment:
Shouldn't these individual steps be proper sentences with a capital letter in the first work and a period at the then?

Reply:
Bullet points don't have to be complete sentences.